content/en/docs/tasks/traffic-management/ingress/ingress-certmgr/index.md

```diff
@@ -22,15 +22,15 @@ $ helm template $HOME/istio-fetch/istio \
   --set global.k8sIngress.enabled=true \
   --set global.k8sIngress.enableHttps=true \
   --set global.k8sIngress.gatewayName=ingressgateway \
-  --set certmanager.enabled=true \
-  --set certmanager.email=mailbox@donotuseexample.com \
   > $HOME/istio-fetch/istio.yaml
 {{< /text >}}
 
 {{< tip >}}
 By default `istio-ingressgateway` will be exposed as a `LoadBalancer` service type. You may want to change that by setting the `gateways.istio-ingressgateway.type` installation option to `NodePort` if this is more applicable to your Kubernetes environment.
 {{< /tip >}}
 
+[Install cert-manager](https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html) if you want to manage certificates automatically.
```

Comment:
Conditionally installing cert-manger "if you ..." sounds wrong given the title "Kubernetes Ingress with Cert-Manager".

Reply:
Also, how about making the "Install Istio" and "Install cert-manager" two steps in a numbered list?